GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/HarvestTemple.cs

```diff
@@ -521,6 +541,34 @@ internal override void EIEvtcParse(ulong gw2Build, FightData fightData, AgentDat
         internal override void ComputeNPCCombatReplayActors(NPC target, ParsedEvtcLog log, CombatReplay replay)
         {
             var knownEffectsIDs = new HashSet<long>();
+            var dragons = new List<int>
+            {
+                (int)ArcDPSEnums.TargetID.TheDragonVoidJormag,
+                (int)ArcDPSEnums.TargetID.TheDragonVoidPrimordus,
+                (int)ArcDPSEnums.TargetID.TheDragonVoidKralkatorrik,
+                (int)ArcDPSEnums.TargetID.TheDragonVoidMordremoth,
+                (int)ArcDPSEnums.TargetID.TheDragonVoidZhaitan,
```

Comment:
Same comment as for the red selection effect.
You can also put that inside DragonBodyVoidAmalgamate case as reds could be dropped while a dragon aspect is not present.
You can use FindActiveOrNextDragonVoid API to find the proper end time.